src/hooks/useDownloadFile.ts

```diff
@@ -0,0 +1,33 @@
+import {useEffect, useState} from 'react';
+
+import {saveAs} from 'file-saver';
+
+type DownloadOptions = {
+  filename?: string;
+};
+
+export const useDownloadFile = (textToCopy: string, options: DownloadOptions) => {
+  const [isProcessed, setProcessed] = useState(false);
+
+  const downloadFile = (content: string, filename: string = 'output.sh') => {
```

Comment:
same question

Reply:
I'm open for suggestions. For know I have the format {execution-name}.sh. If you have any preference please let me know.